about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>About &mdash; Saejin Kwak Tanguay</title>
+    <link rel="stylesheet" href="./stylesheets/style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="navbar">
+          <a href="mailto:saejinkwak@gmail.com">Contact</a>
+          <a href="./blog.html">Blog</a>
+          <a href="./portfolio.html">Portfolio</a>
+          <a href="./courses.html">Courses & Research</a>
+          <a href="./about.html">About</a>
+          <a href="./index.html">Home</a>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <h2>Academic Career</h2>
+      <p>Stuff</p>
+      <h2>From Education to Engineering</h2>
+      <p>More stuff</p>
+      <img src="./images/STARS.JPG" class="wide"/>
```

Comment:
Just a note that all your `img` tags should have `alt` attributes.  [It's for accessibility.](https://www.w3.org/WAI/tutorials/images/)